src/convertToObject.js

```diff
@@ -6,7 +6,28 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const declarations = sourceString.trim().split(';');
+
+  const stylesObj = {};
+
+  declarations.forEach((declaration) => {
+    const parts = declaration.split(':');
+
+    if (parts.length === 2) {
+      const property = parts[0].trim();
+      let value = parts[1].trim();
+
+      if (value.includes('!important')) {
+        value = value.replace('!important', '').trim() + ' !important';
```

Comment:
The handling of '!important' is correct, but ensure that this behavior is intended. The code moves '!important' to the end of the value, which is generally correct for CSS parsing.